src/modules/commander/Commander.cpp

```diff
@@ -3662,10 +3662,14 @@ void Commander::battery_status_check()
 		// Try to trigger RTL
 		if (main_state_transition(_status, commander_state_s::MAIN_STATE_AUTO_RTL, _status_flags,
 					  _internal_state) == TRANSITION_CHANGED) {
-			mavlink_log_emergency(&_mavlink_log_pub, "Flight time low, returning to land");
+			mavlink_log_emergency(&_mavlink_log_pub, "Remaining flight time low, returning to land\t");
+			events::send(events::ID("commander_remaining_flight_time_rtl"), events::Log::Emergency,
```

Comment:
Isn't Emergency a bit too extreme? For comparison, the legacy "RTL due to critical battery" triggers an event with:
` events::Log::Critical, events::LogInternal::Info`

Reply:
You're completely right, thanks! I compared to https://github.com/PX4/PX4-Autopilot/blob/01db98fccf7944a98f5ee8b14693a58d1fca34be/src/modules/commander/state_machine_helper.cpp#L1143 which should have the same severity and it's like you describe externally `Critical`, internally `Info`.

I corrected the commit directly.